lightning/src/chain/channelmonitor.rs

```diff
@@ -635,25 +635,35 @@ pub enum Balance {
 		/// [`Balance::MaybeTimeoutClaimableHTLC`] with their
 		/// [`Balance::MaybeTimeoutClaimableHTLC::outbound_payment`] flag set, as well as any dust
 		/// HTLCs which would otherwise be represented the same.
+		///
+		/// This amount (rounded up to a whole satoshi value) will not be included in `amount_satoshis`.
 		outbound_payment_htlc_rounded_msat: u64,
 		/// The amount of millisatoshis which has been burned to fees from HTLCs which are outbound
 		/// from us and are related to a forwarded HTLC. This is the sum of the millisatoshis part
 		/// of all HTLCs which are otherwise represented by [`Balance::MaybeTimeoutClaimableHTLC`]
 		/// with their [`Balance::MaybeTimeoutClaimableHTLC::outbound_payment`] flag *not* set, as
 		/// well as any dust HTLCs which would otherwise be represented the same.
+		///
+		/// This amount (rounded up to a whole satoshi value) will not be included in `amount_satoshis`.
 		outbound_forwarded_htlc_rounded_msat: u64,
 		/// The amount of millisatoshis which has been burned to fees from HTLCs which are inbound
 		/// to us and for which we know the preimage. This is the sum of the millisatoshis part of
 		/// all HTLCs which would be represented by [`Balance::ContentiousClaimable`] on channel
-		/// close, but who's current value is included in
+		/// close, but whose current value is included in
 		/// [`Balance::ClaimableOnChannelClose::amount_satoshis`], as well as any dust HTLCs which
 		/// would otherwise be represented the same.
+		///
+		/// This amount (rounded up to a whole satoshi value) will not be included in the counterparty's
+		/// `amount_satoshis` or equivalent thereof.
 		inbound_claiming_htlc_rounded_msat: u64,
 		/// The amount of millisatoshis which has been burned to fees from HTLCs which are inbound
 		/// to us and for which we do not know the preimage. This is the sum of the millisatoshis
 		/// part of all HTLCs which would be represented by [`Balance::MaybePreimageClaimableHTLC`]
 		/// on channel close, as well as any dust HTLCs which would otherwise be represented the
 		/// same.
+		///
+		/// This amount (rounded up to a whole satoshi value) will not be included in the counterparty's
+		/// `amount_satoshis` or equivalent thereof.
```

Comment:
What does "or equivalent thereof" refer to?

Reply:
I suppose it's not worth adding that part to refer to whatever other implementations call that. Will remove.